woocommerce-gateway-paypal-express-checkout.php

```diff
@@ -47,3 +47,19 @@ function wc_gateway_ppec() {
 }
 
 wc_gateway_ppec()->maybe_run();
+
+/**
+ * Adds the WooCommerce Inbox option on plugin activation
+ *
+ * @since 2.1.2
+ */
+if ( ! function_exists( 'add_woocommerce_inbox_variant' ) ) {
+	function add_woocommerce_inbox_variant() {
+		$option_name = 'woocommerce_inbox_variant_assignment';
```

Comment:
Similar to what was mentioned here: https://github.com/woocommerce/woocommerce-paypal-payments/pull/160#pullrequestreview-655255648
This should be `$option`.